apps/balancer-tools/src/app/poolsimulator/(components)/AnalysisPoolParamsForm.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import { useEffect } from "react";
+import { useForm } from "react-hook-form";
+
+import { Select, SelectItem } from "#/components/Select";
+import { Form, FormLabel } from "#/components/ui/form";
+import { usePoolSimulator } from "#/contexts/PoolSimulatorContext";
+
+export default function AnalysisPoolParamsForm() {
+  const {
+    initialData,
+    customData,
+    setAnalysisTokenByIndex,
+    analysisToken,
+    setCurrentTabTokenByIndex,
+  } = usePoolSimulator();
+
+  const form = useForm({
+    mode: "onChange",
+  });
+
+  const {
+    setValue,
+    getValues,
+    formState: { errors },
+  } = form;
+
+  const onSubmit = () => {
+    if (Object.keys(errors).length) return;
+  };
+
+  useEffect(() => {
+    setAnalysisTokenByIndex(0);
+    setCurrentTabTokenByIndex(1);
```

Comment:
Shouldn't these be the default?

Reply:
`setAnalysisTokenByIndex()` is not a normal state function but this:

```
 function setAnalysisTokenByIndex(index: number) {
    const token = initialData.tokens[index];
    if (token) setAnalysisToken(token);
  }
```

And `setAnalysisToken()` by default is not the data we imported it. Does that make sense to you?